src/libs/nav.ts

```diff
@@ -73,13 +73,19 @@ export const getPath = (name: string, params?: Record<string, any>): string => {
 /**
  * alias for getPath()
  */
-export const getLink = (name: string, params?: Record<string, any>) => `#${getPath(name, params).slice(1)}`; // slice off leading slash
+export const getLink = (name: string, params?: Record<string, any>) =>
+  `#${getPath(name, params).slice(1)}${
+    params?.queryParams ? qs.stringify(params.queryParams, { addQueryPrefix: true }) : ''
```

Comment:
using `queryParams` as the key to align with how they can be accessed in components https://github.com/DataBiosphere/terra-ui/blob/8062b76eb3f108381be0a5f5ee3cf85a4cff8518/src/libs/nav.ts#L135